ttnn/cpp/ttnn/tensor/layout/size.cpp

```diff
@@ -0,0 +1,61 @@
+// SPDX-FileCopyrightText: © 2024 Tenstorrent AI ULC
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#include "size.hpp"
+#include <ostream>
+
+namespace tt::tt_metal {
+
+Size::Size(size_t height, size_t width)
+    : m_height(height), m_width(width) {}
+
+Size::Size(const std::pair<size_t, size_t>& size)
+    : Size(size.first, size.second) {}
+
+Size::Size(const std::array<size_t, 2>& size)
+    : Size(size[0], size[1]) {}
+
+Size Size::operator/(const Size& rhs) const {
+    return Size(m_height / rhs.m_height, m_width / rhs.m_width);
+}
```

Comment:
Is this operator used? I think it can be misleading. To me, size divided by size is something that is not necessarily size. It's some multiple. Also, I think dividing up is actually more useful for most cases.

Reply:
I agree. Will remove.  
Thank you!

I went with this thinking that it is useful to know how many parts fit vertically and how many fits horizontally. Looked at it again and checked APIs for a similar type in other libs and I think this is a bad API.